src/style.css

```
@@ -1,3 +1,92 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap';
+
 body {
   margin: 0;
+  font-family: ' Roboto ', sans-serif;
```

Move font styles to the html selector
```suggestion
  font-family: Roboto, sans-serif;
```